common/Hooks/prepare-commit-msg.py

```diff
@@ -21,6 +23,19 @@
 
 def commit_scope(commit_dir: str) -> str:
     if os.path.exists(os.path.join(commit_dir, 'package.yml')):
+
+        recipe_diff_result = subprocess.run(['git', 'diff', '-U0', '--staged', '--word-diff',
+                                            os.path.join(commit_dir, 'package.yml')],
+                                            stdout=subprocess.PIPE)
+        if "version" in recipe_diff_result.stdout.decode('utf-8'):
+            with open(os.path.join(commit_dir, 'package.yml')) as recipe:
+                try:
+                    data = yaml.safe_load(recipe)
+                    version = data['version']
+                    return os.path.basename(commit_dir) + ': Update to ' + str(version)
+                except yaml.YAMLError as e:
+                    print(e)
```

Comment:
Should this exception ever occur? I feel like it should never happen, in which case it's fine not to catch the exception:

```suggestion
                    version = yaml.safe_load(recipe)['version']
                    return os.path.basename(commit_dir) + ': Update to ' + str(version)
```